PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -139,9 +159,16 @@ struct HfCandidateCreator2Prong {
     ccdb->setLocalObjectValidityChecking();
     lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbPathLut));
     runNumber = 0;
+
+    /// collision monitoring
+    hCollisions->GetXaxis()->SetBinLabel(1, "All collisions");
+    hCollisions->GetXaxis()->SetBinLabel(2, "Centrality ok");
+    hCollisions->GetXaxis()->SetBinLabel(3, "Centrality + sel8 ok");
+    hCollisions->GetXaxis()->SetBinLabel(4, "Centrality + sel8 + posZ ok");
+    hCollisions->GetXaxis()->SetBinLabel(5, "Centrality + sel8 + posZ + TF border ok");
```

Comment:
It would be better to sync the bin numbers with the enum values. The `SetBinLabel` part could be done by a helper function in the utils file.